anecdote/metadata.csv

```diff
@@ -0,0 +1,31 @@
+anecdote.feedback.Freshdesk,count,,,,Counts the total number of support tickets from Freshdesk.,1,anecdote,feedback.Freshdesk,
+anecdote.feedback.Google_Maps,count,,,,Counts the total number of reviews from Google Maps.,1,anecdote,feedback.Google_Maps,
+anecdote.feedback.G2,count,,,,Counts the total number of reviews from G2.,1,anecdote,feedback.G2,
+anecdote.feedback.Trustpilot,count,,,,Counts the total number of reviews from Trustpilot.,1,anecdote,feedback.Trustpilot,
+anecdote.feedback.Reddit,count,,,,Counts the total number of mentions from Reddit.,1,anecdote,feedback.Reddit,
+anecdote.feedback.hubspot,count,,,,Counts the total number of customer feedback entries from HubSpot.,1,anecdote,feedback.hubspot,
+anecdote.feedback.typeform,count,,,,Counts the total number of responses from Typeform surveys.,1,anecdote,feedback.typeform,
+anecdote.feedback.Delighted,count,,,,Counts the total number of feedback entries from Delighted.,1,anecdote,feedback.Delighted,
+anecdote.feedback.Discord,count,,,,Counts the total number of messages from Discord.,1,anecdote,feedback.Discord,
+anecdote.feedback.SurveyMonkey,count,,,,Counts the total number of responses from SurveyMonkey.,1,anecdote,feedback.SurveyMonkey,
+anecdote.feedback.Steam,count,,,,Counts the total number of reviews from Steam.,1,anecdote,feedback.Steam,
+anecdote.feedback.App_Store.bug,count,,,,Counts the total number of bug-related feedback entries from the Apple Store.,1,anecdote,feedback.App_Store.bug,
+anecdote.feedback.Google_Play.bug,count,,,,Counts the total number of bug-related feedback entries from Google Play Store.,1,anecdote,feedback.Google_Play.bug,
+anecdote.feedback.custom-csv.bug,count,,,,Counts the total number of bug-related feedback entries from custom CSV sources.,1,anecdote,feedback.custom-csv.bug,
+anecdote.feedback.custom-parquet.bug,count,,,,Counts the total number of bug-related feedback entries from custom Parquet sources.,1,anecdote,feedback.custom-parquet.bug,
+anecdote.feedback.Twitter.bug,count,,,,Counts the total number of bug-related tweets from Twitter.,1,anecdote,feedback.Twitter.bug,
+anecdote.feedback.Intercom.bug,count,,,,Counts the total number of bug-related customer interactions from Intercom.,1,anecdote,feedback.Intercom.bug,
+anecdote.feedback.SendBird.bug,count,,,,Counts the total number of bug-related messages from SendBird platforms.,1,anecdote,feedback.SendBird.bug,
+anecdote.feedback.gorgias.bug,count,,,,Counts the total number of bug-related customer support tickets from Gorgias.,1,anecdote,feedback.gorgias.bug,
+anecdote.feedback.Zendesk_Support.bug,count,,,,Counts the total number of bug-related support tickets from Zendesk Support.,1,anecdote,feedback.Zendesk_Support.bug,
+anecdote.feedback.Freshdesk.bug,count,,,,Counts the total number of bug-related support tickets from Freshdesk.,1,anecdote,feedback.Freshdesk.bug
```

Comment:
From github: "It looks like row 31 should actually have 10 columns, instead of 9"
```suggestion
anecdote.feedback.Freshdesk.bug,count,,,,Counts the total number of bug-related support tickets from Freshdesk.,1,anecdote,feedback.Freshdesk.bug,
```